PlayBook/src/SUMMARY.md

```diff
@@ -16,6 +16,14 @@
     -   [Importing & Exporting Keymaps](./keymapping/import_export_keymaps.md)
     -   [Common Issues](./keymapping/common_issues.md)
 
+# Account Manager
+
```

Comment:
A readme is needed for this category, as the ones in https://github.com/PlayCover/PlayBook/pull/56.